src/components/KSelect/KSelect.vue

```diff
@@ -480,7 +480,7 @@ const value = computed({
     return props.modelValue
   },
   set(newValue: string | number): void {
-    const item = selectItems.value.filter((item: SelectItem) => item.value === newValue)
+    const item = selectItems.value?.filter((item: SelectItem) => item.value === newValue)
     if (item.length) {
```

Comment:
if `item` can be undefined, then the if statement below needs to optionally chain checking for the `length`

Reply:
fixed